exercises/practice/anagram/anagram_test.go

```diff
@@ -8,10 +8,16 @@ import (
 
 var nonAsciiTestCases = []anagramTest{
 	{
-		description: "detects non-ascii anagrams",
-		subject:     "你好，世界",
-		candidates:  []string{"世界，你好", "hello, 世界", "世界, 你好"},
-		expected:    []string{"世界，你好"},
+		description: "detects unicode anagrams",
+		subject:     "ΑΒΓ",
+		candidates:  []string{"ΒΓΑ", "ΒΓΔ", "γβα"},
+		expected:    []string{"ΒΓΑ", "γβα"},
+	},
+	{
+		description: "eliminates misleading unicode anagrams",
```

Comment:
Is this test needed? What is it trying to test?

I get that `Α` and `A` are different runes, despite looking similar. But I can't think of a solution that would pass the other tests and fail this one, but maybe I'm overlooking something.

Reply:
@andrerfcsantos I took these two test cases from old Exercism test cases when Anagram wasn't limited to ASCII. I will remove this one as it provides little value to the exercise.